reference/sectionC_interpretation.inc

```diff
@@ -232,6 +238,8 @@ yocto    −24
 Component reference
 -------------------
 
+A "component reference" refers to the :code:`component_ref` attribute used in :code:`import component` items, to the :code:`component_ref` elements which occurs as children of :code:`encapsulation` items, and to the :code:`component_1` and :code:`component_2` attributes used in :code:`connection` items.
```

Comment:
See remarks in https://github.com/cellml/cellml-specification/pull/87/files#r398161216.

Reply:
Done in cff1dc9